orc8r/cloud/helm/orc8r/charts/certs/templates/admin-operator-certificate.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.create }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name }}-admin-operator-certificate
```

Comment:
Side note for a future PR: we'll still need to inject this default cert into the certifier service https://magma.github.io/magma/docs/next/orc8r/deploy_install#create-an-orchestrator-admin-user

Though perhaps we can invert this requirement -- instead of it being a manual step during installation, the certifier service can instead proactively check for the default-admin-operator-tls secret and pull it in if it exists

Reply:
I will look into it next once this PR has merged.